src/main/resources/application.yml

```diff
@@ -65,10 +65,15 @@ cors:
     - "https://www.nav.no/familie/alene-med-barn/ettersending"
 
 no.nav.security.jwt:
-  issuer.selvbetjening:
-    discoveryurl: ${LOGINSERVICE_IDPORTEN_DISCOVERY_URL}
-    accepted_audience: ${LOGINSERVICE_IDPORTEN_AUDIENCE}
-    cookie_name: selvbetjening-idtoken
+  issuer:
+    selvbetjening:
+      discoveryurl: ${LOGINSERVICE_IDPORTEN_DISCOVERY_URL}
+      accepted_audience: ${LOGINSERVICE_IDPORTEN_AUDIENCE}
+      cookie_name: selvbetjening-idtoken
+    tokenx:
+      discovery-url: ${TOKEN_X_WELL_KNOWN_URL}
+      accepted-audience: ${TOKEN_X_CLIENT_ID}
+      cookie_name: tokenx-token
```

Comment:
Trenger vi cookie_name her?

Reply:
For lokal kjøring så tror jeg det er denne som brukes. Men kanskje den burde vært i application-local ? 🤔